tracks.go

```diff
@@ -0,0 +1,46 @@
+package spotify
+
+import (
+	"strconv"
+	"time"
+)
+
+type Tracks struct {
+	ItemsMeta
+	Items []Track
+}
+
+type Track struct {
+	AddedAt time.Time `json:"added_at"`
+	AddedBy Meta      `json:"added_by"`
+	IsLocal bool      `json:"is_local"`
+	Track   TrackInfo
+	URI     string
+}
+
+type TrackInfo struct {
+	Meta
+	Album            Album
+	Artists          []Artist
+	AvailableMarkets []string  `json:"available_markets"`
+	DiscNumber       int       `json:"disc_number"`
+	Duration         *Duration `json:"duration_ms"`
```

Comment:
Changing this from an `int` to a `*Duration` is a breaking change... it's ok now, but it should definitely be avoided in the future if anyone else uses this project, JSYK.

Reply:
Agreed. In this case, `Duration` as the type makes _a lot_ of sense, which made the breaking change worth it.